### PR TITLE
Article on kubectl top pod/node

### PR DESCRIPTION
**Pull request by ankit01-oss**

### CODE REVIEW

**Comment by prashant-shahi** [comment]
@ankit01-oss `kubectl top` commands don't work without metrics-server. We need to mention the need of it with reference to the [metrics-server repository](https://github.com/kubernetes-sigs/metrics-server).

Some vendors include it by default while most don't.
For example, Google Kubernetes Engine clusters include a Metrics Server deployment by default, whereas Amazon Elastic Kubernetes Service clusters do not.

**Comment by ankit01-oss** [comment]
@prashant-shahi we have mentioned it here. 

Any other gaps that you see in the article? Would be great if you can review.

Screenshot 2022-06-09 at 5 17 44 PM

**Comment by pranay01** [comment]
will wait for any more comments by @prashant-shahi before merging this. Prashant let me know when its good to merge

File: blog/2022-06-09-kubectl-top.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+## What is a pod in Kubernetes?
+
+Kubernetes is meant to orchestrate the management of containers. Pods are the first level of abstraction that it provides over containers.
+
+Pods are groups of one or more containers with shared resources like storage and networks. They are used as units of replication in cases where applications need to be scaled up or down. 
+
+## What is a node in Kubernetes?
+
+They are the smallest unit of computing hardware in Kubernetes clusters. They are single machines where applications run, which could be a physical server or a virtual machine. They help in load distribution for cases where there is an error or failure in a node.
```
Comment (prashant-shahi) [inline blog/2022-06-09-kubectl-top.md]: > smallest unit of computing hardware

K8s nodes are usually considered as "bulky" or something big.
And usually "smallest deployable units of computing" is the term used for Pods.
We could perhaps rephrase the first sentence.

Comment (ankit01-oss) [reply blog/2022-06-09-kubectl-top.md]: @prashant-shahi can you directly make the changes and commit?

File: blog/2022-06-09-kubectl-top.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+- Memory%
+It is also displayed only for nodes, and it stands for total memory usage percentage of that node.
+
+## Using `kubectl top pod` command
+
+Running the `kubectl top pod` command displays the metrics about pods from the `default` namespace which looks like this:
+
+```docker
+NAME	                             CPU (CORES)	MEMORY (BYTES)
+nginx                              3m           1Mi
```
Comment (prashant-shahi) [inline blog/2022-06-09-kubectl-top.md]: indentation in this code block is not right

**Comment by ankit01-oss** [comment]
@pranay01 fixed in new commit. You can merge.

**Pull request merged**